[2024-07-17T09:26:55.646+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-17T09:26:55.694+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Postgres_dag.insert_into_table scheduled__2024-07-10T00:00:00+00:00 [queued]>
[2024-07-17T09:26:55.704+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Postgres_dag.insert_into_table scheduled__2024-07-10T00:00:00+00:00 [queued]>
[2024-07-17T09:26:55.705+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-07-17T09:26:55.780+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2024-07-10 00:00:00+00:00
[2024-07-17T09:26:55.789+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=3120) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-17T09:26:55.791+0000] {standard_task_runner.py:63} INFO - Started process 3128 to run task
[2024-07-17T09:26:55.788+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Postgres_dag', 'insert_into_table', 'scheduled__2024-07-10T00:00:00+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres.py', '--cfg-path', '/tmp/tmp57f_v_o9']
[2024-07-17T09:26:55.792+0000] {standard_task_runner.py:91} INFO - Job 38: Subtask insert_into_table
[2024-07-17T09:26:55.940+0000] {task_command.py:426} INFO - Running <TaskInstance: Postgres_dag.insert_into_table scheduled__2024-07-10T00:00:00+00:00 [running]> on host 67c97fe0936f
[2024-07-17T09:26:56.218+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Rayzo' AIRFLOW_CTX_DAG_ID='Postgres_dag' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2024-07-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-07-10T00:00:00+00:00'
[2024-07-17T09:26:56.222+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-17T09:26:56.258+0000] {sql.py:276} INFO - Executing:  
        insert into drag_runs(dt,dag_id) values('2024-07-10', 'Postgres_dag')
        
[2024-07-17T09:26:56.278+0000] {base.py:84} INFO - Using connection ID 'Postgres_con' for task execution.
[2024-07-17T09:26:56.299+0000] {base.py:84} INFO - Using connection ID 'Postgres_con' for task execution.
[2024-07-17T09:26:56.315+0000] {sql.py:487} INFO - Running statement:  
        insert into drag_runs(dt,dag_id) values('2024-07-10', 'Postgres_dag')
        , parameters: None
[2024-07-17T09:26:56.320+0000] {sql.py:496} INFO - Rows affected: 1
[2024-07-17T09:26:56.392+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-17T09:26:56.407+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Postgres_dag, task_id=insert_into_table, run_id=scheduled__2024-07-10T00:00:00+00:00, execution_date=20240710T000000, start_date=20240717T092655, end_date=20240717T092656
[2024-07-17T09:26:56.499+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-17T09:26:56.517+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-17T09:26:56.597+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
